CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Members of the CES administrator group receive project admin permissions for new projects (#3)
+- CES-ADMIN group can be enabled to administer all projects using the key `amend_projects_with_ces_admin_permissions`
```

Comment:
I see there are some different conventions on naming the CES admin group?
- here it is CES-ADMIN
- in the documentation below it is called `CES_ADMIN`
- in script loggings it is CES-Admin

Also both lines basically say the same thing. Maybe join them and remove redundancy?

Reply:
My intention here was to clarify that the change will only apply for new projects. In my perspective the config key has more or less a migration purpose, which are two different things, i guess. The correct version should be `CES_ADMIN` i will correct that.